setup.py

```diff
@@ -21,6 +21,7 @@
     "docs": [
         "sphinx>=5.3.0",
         "sphinx_rtd_theme>=1.0.0",
+        "sphinx-autobuild>=2024.2.4",
```

Comment:
You'll need to dop the `sphinx-autobuild` version back a bit:

`ERROR: Ignored the following versions that require a different python version: 2024.2.4 Requires-Python >=3.9`